lumibot/data_sources/alpaca_data.py

```diff
@@ -184,7 +201,9 @@ def get_barset_from_api(self, asset, freq, limit=None, end=None, start=None, quo
         if not start:
             if str(freq) == "1Min":
                 if datetime.now().weekday() == 0:  # for Mondays as prior days were off
-                    loop_limit = limit + 4896  # subtract 4896 minutes to take it from Monday to Friday, as there is no data between Friday 4:00 pm and Monday 9:30 pm causing an incomplete or empty dataframe
+                    loop_limit = (
+                        limit + 4896
+                    )  # subtract 4896 minutes to take it from Monday to Friday, as there is no data between Friday 4:00 pm and Monday 9:30 pm causing an incomplete or empty dataframe
```

Comment:
I don't like this expanding out with the ( ) where in all other situations you are removing them.  For long comments, simply move the comment to be above the code line rather than adjacent.